hbase-server/src/main/java/org/apache/hadoop/hbase/util/hbck/OfflineMetaRepair.java

```diff
@@ -17,49 +17,17 @@
  */
 package org.apache.hadoop.hbase.util.hbck;
 
-import java.io.IOException;
-
-import org.apache.commons.lang3.StringUtils;
 import org.apache.yetus.audience.InterfaceAudience;
 import org.apache.yetus.audience.InterfaceStability;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.fs.Path;
-import org.apache.hadoop.hbase.HBaseConfiguration;
 import org.apache.hadoop.hbase.HBaseInterfaceAudience;
-import org.apache.hadoop.hbase.util.FSUtils;
-import org.apache.hadoop.hbase.util.HBaseFsck;
-import org.apache.hadoop.io.MultipleIOException;
 
 /**
- * This code is used to rebuild meta off line from file system data. If there
- * are any problem detected, it will fail suggesting actions for the user to do
- * to "fix" problems. If it succeeds, it will backup the previous hbase:meta and
- * -ROOT- dirs and write new tables in place.
- *
- * This is an advanced feature, so is only exposed for use if explicitly
- * mentioned.
- *
- * hbase org.apache.hadoop.hbase.util.hbck.OfflineMetaRepair ...
+ * We've deprecated this tool in hbase-2+ because it destroyed the hbase2 meta table.
```

Comment:
Could you please move this to the deprecated Javadoc annotation, stating the earliest version it was deprecated and in which version it should be removed?